src/app/components/text-expand/index.html

```diff
@@ -0,0 +1,53 @@
+<sky-demo-page title="Text Expand">
+  <sky-demo-page-summary>
+    <p>
+      The Text Expand component truncates long text with an ellipsis and a link that allows the user to fully expand the text.
+      If the text length falls below the specified threshold then no action is taken. The 'See more' link will expand the
+      text inline if the length of the text is below the specified expanded length limit and if the number of newlines in
+      the text is below the specified expanded newline limit, otherwise the link will open the text in a modal view.
+    </p>
+    <p>
+      Note that collapsed text will have newlines removed. Also, if one or more newlines are detected, the text is automatically
+      collapsed regardless of the total length of the text.
+    </p>
+    <p>
+      The Text Expand Repeater directive truncates a list of repeater items and will initially display a set number of items. Any items over the set maximum limit are hidden until the user elects to expand the list.
+    </p>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties sectionHeading="Text expand properties">
+    <sky-demo-page-property propertyName="text">
+      The text to truncate
+    </sky-demo-page-property>
+    <sky-demo-page-property propertyName="maxLength" defaultValue="200" isOptional="true">
+      The number of characters to show before truncating the text. The directive will attempt to look back up to 10 characters
+      for a space and truncate there in order to avoid truncating in the middle of a word.
```

Comment:
Is there a better way to reword this? 
> "The number of characters to show before truncating the text. The directive will attempt to look back up to 10 characters for a space and truncate there in order to avoid truncating in the middle of a word."

How about something like...

> "The number of characters that are shown before truncation. The directive will attempt to avoid truncating in the middle of a word by allowing for a 10 character tolerance."